index.d.ts

```diff
@@ -2711,7 +2711,6 @@ declare namespace Shopify {
     enabled_presentment_currencies: string[];
     email: string;
     finances: boolean;
-    force_ssl: boolean;
```

Comment:
According to https://shopify.dev/api/admin/rest/reference/store-properties/shop the property was not removed but only deprecated. Its value is always `true`.

Reply:
ok so can we make it not mandatory

Reply:
Yes, please do that.

Reply:
done